packages/kui-builder/lib/configure.js

```diff
@@ -49,6 +49,40 @@ const readIndex = options =>
     })
   })
 
+/**
+ * Unify the given settings of configurations
+ *
+ */
+const unify = (setting1, setting2) => {
+  const uniqe = arr => {
+    return arr.filter((element, index) => arr.indexOf(element) === index)
+  }
```

Comment:
unique() is for webpack-watch. webpack-watch calls configure a couple of times. With my unify, webpack-watch will keep appending duplicated element to array, and error out for duplicated string value.